VRCFaceTracking/Params/Expressions/Legacy/Lip/UnifiedSRanMapper.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace VRCFaceTracking.Params.Lip
+{
+    internal static class UnifiedSRanMapper
```

Comment:
Just want to make sure that the SRanipal conversion works exactly as intended or works as expected for everyone? SRanipal should work exactly as it did before the parameter overhaul (VRCFT 4.0), and it should be better suited to handle Unified Expressions.